app/views/pages/home.html.erb

```diff
@@ -1,5 +1,7 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<h2>Our new homepage!</h2>
```

Comment:
I think it should be an `h1`, what do you think?

Reply:
You're right. On it!

Reply:
Done. @scastiel could you validate?